CompilerSource/compiler/components/parse_and_link.cpp

```diff
@@ -95,7 +95,7 @@ int lang_CPP::compile_parseAndLink(EnigmaStruct *es,parsed_script *scripts[], ve
       // Keep a parsed record of this timeline
       tline_lookup[es->timelines[i].name].push_back(tlines.back());
       parser_main(newcode, &tlines.back()->pev, script_names);
-      edbg << "Parsed `" << es->timelines[i].name <<", moment: " <<es->timelines[i].moments[j].stepNo << "': " << tlines.back()->obj.locals.size() << " locals, " << tlines.back()->obj.globals.size() << " globals" << flushl;
+      edbg << "Parsed `" << es->timelines[i].name <<", moment: " <<es->timelines[i].moments[j].stepNo << "': " << (int)(tlines.back()->obj.locals.size()) << " locals, " << (int)tlines.back()->obj.globals.size() << " globals" << flushl;
```

Comment:
That actually should not be necessary because I already overloaded it for ``size_t`` before I thought.
https://github.com/enigma-dev/enigma-dev/commit/5b99151173b1dad9acb059f9e99a75974801d42b

Hrm... actually looks like I missed ``size_t`` for ideprint. Follow the changes I made in that file to overload ideprint for ``size_t``.